src/CMakeLists.txt

```diff
@@ -464,7 +464,9 @@ if(NOT CEPH_BUILD_VIRTUALENV)
 endif()
```

Comment:
These changes are needed to compile ceph-disk, right ?

Reply:
Just checked this is the same as in https://github.com/starlingx-staging/stx-ceph/blob/ddfb3f76cc29f499fa13730e8ff6335d80cdc3b2/src/CMakeLists.txt#L773